code42_connector.py

```diff
@@ -333,7 +333,11 @@ def _handle_get_user_profile(self, param, action_result):
         user_id = self._get_user_id(username)
         response = self._client.detectionlists.get_user_by_id(user_id)
         all_tags = response.data.get("riskFactors", [])
+        all_cloud_usernames = response.data.get("cloudUsernames", [])
         response["riskFactors"] = _convert_to_obj_list(all_tags, "tag")
+        response["cloudUsernames"] = _convert_to_obj_list(
```

Comment:
Also did this for cloud usernames in case some day we add the add/remove cloud usernames actions to this app